cumulus_api/main.py

```diff
@@ -1,50 +1,63 @@
-import json
 import logging
 import os
-from configparser import ConfigParser, SectionProxy
+import re
+from configparser import ConfigParser
 from json.decoder import JSONDecodeError
-from typing import Union
+from types import SimpleNamespace
 
-import requests
+from requests import Session
 
 from .cumulus_token import CumulusToken
 
 
 # pylint: disable=too-many-public-methods
 class CumulusApi:
-    def __init__(self, use_os_env: bool = True, config_path: str = None, token: str = None):
+    def __init__(self, token=None, config_file=None):
```

Comment:
`config_file` should remain `config_path` 
Updating will require py-cumulus-api users to update this variable name if already set within class instantiation and the update doesn't provide any additional context/functionality beyond a name change